config/kubernetes-sigs/sig-cluster-lifecycle/teams.yaml

```diff
@@ -307,6 +307,7 @@ teams:
     - justinsb
     - kkeshavamurthy
     - luxas
+    - mboersma
```

Comment:
could you please update the whole list? I see several names (include myself) that shouldn't be in here anymore

Reply:
Updated so both lists are just the three current maintainers.